packages/date-picker/test/basic.test.js

```diff
@@ -93,6 +93,13 @@ describe('basic features', () => {
     expect(datepicker.hasAttribute('focused')).to.be.true;
   });
 
+  it('should close the dropdown on Esc when on fullscreen', async () => {
```

Comment:
Is this test related to the stop propagation changes? I tried to revert the changes and no test really failed.

Reply:
Thanks. Yes, it seems like we need to focus buttons manually.

Reply:
Updated the tests.